File: visual_basic/integracion_con_base_de_datos/Mostrar_contactos/conexion.vb
Imports MySql.Data.MySqlClient
Module conexion
    Public Function ObtenerConexion() As MySqlConnection
        Dim cadenaConexion As String = "server=localhost;user id=root;password=;database=mostrar_esos_contactos"
        Return New MySqlConnection(cadenaConexion)
    End Function
End Module

File: python/eliminar_contacto/conexion.py
import mysql.connector
def conectar():
    try:
        conexion = mysql.connector.connect(
            host="localhost",
            user="root",
            password="",
            database="eliminar_contacto"
        )
        return conexion
    except mysql.connector.Error as e:
        print("Error al conectar:", e)
        return None